docs/sources/panels-visualizations/visualizations/table/index.md

```diff
@@ -226,8 +228,14 @@ If you want to show the number of rows in the dataset instead of the number of v
 [calculations]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/query-transform-data/calculation-types"
 [calculations]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/visualizations/panels-visualizations/query-transform-data/calculation-types"
 
-[time series to table]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/query-transform-data/transform-data#time-series-to-table-transform"
-[time series to table]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/query-transform-data/transform-data#time-series-to-table-transform"
+[Time series to table transformation]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/query-transform-data/transform-data#time-series-to-table-transform"
+[Time series to table transformation]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/query-transform-data/transform-data#time-series-to-table-transform"
+
+[Time series panel]: "docs/grafana/" --> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/visualizations/time-series/
+[Time series panel]: "docs/grafana-cloud/" --> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/visualizations/time-series/
+
+[color scheme]: "docs/grafana/" --> docs/grafana/<GRAFANA VERSION>/panels-visualizations/configure-standard-options#color-scheme
+[color scheme]: "docs/grafana-cloud" -> docs/grafana/<GRAFANA VERSION>/panels-visualizations/configure-standard-options#color-scheme
```

Comment:
There are some formatting errors here. I've also swapped the Cloud links to link to Cloud docs and future-proofed the version interpolation syntax.
```suggestion
[Time series panel]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/panels-visualizations/visualizations/time-series"
[Time series panel]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/visualizations/panels-visualizations/visualizations/time-series"

[color scheme]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/panels-visualizations/configure-standard-options#color-scheme"
[color scheme]: "/docs/grafana-cloud -> /docs/grafana-cloud/visualizations/panels-visualizations/configure-standard-options/#color-scheme"
```

Reply:
Thanks for the changes Isabel!